### DISCUSSION TITLE
Việc làm

### DISCUSSION THREAD

**Discussion by Coder-LabVN** [OngDev/community#232] [Q&A]
Em muốn làm thêm những công việc như viết code, nhưng ban thân là một người chưa biết gì thì có job nào có liên quan đến việc này ngoài giờ hành chính không anh! Và mình có cần học những chứng chỉ để hỗ trợ, có kiến thức căn bản không ạ! Và nên học loại nào để đáp ứng được nhu cầu thị trường và bản thân (như là làm thêm ngoài khung giờ hành chính, hoac có thể làm remote)
Phương tiện máy tính cần tiêu chuẩn gì để có thể làm được.
Nhờ anh giải đáp, xin cảm ơn!

**Comment by lucthienphong1120** [comment]
bạn có thể tìm 1 số job part-time hay freelance
hay đơn giản kỹ năng chưa nhiều thì đi làm bài thuê
